packages/web-components/src/components/va-number-input/va-number-input.scss

```diff
@@ -6,12 +6,19 @@
   display: block;
 }
 
+.usa-sr-only {
+  position: absolute !important;
+  left: -999em !important;
+  right: auto !important;
+}
+
```

Comment:
Issue noticed from Chromatic

Had issues with `.usa-sr-only` styles being overridden by the styles below, so added some `shame.css` worthy code. Figured since this component is on its last legs it would be ok.